setup.py

```diff
@@ -116,6 +116,11 @@ def read_version_py(infname):
     # know works on Python 2.7.
     "eliot ~= 1.7",
 
+    # Pyrsistent 0.17.0 (which we use by way of Eliot) has dropped
+    # Python 2 entirely; stick to the version known to work for us.
+    # XXX: drop this bound: https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3404
+    "pyrsistent <= 0.16.0",
```

Comment:
It could be *slightly* better to say `< 0.17.0` in case they release a bugfix or similar (0.16.1)..

Reply:
Thank you!  Updated in e893254.